### ISSUE TITLE
Distribute Betty Edwards slide content

### ISSUE THREAD

**Issue by whilei** [webappzero/cr17-to-zero#70]
Across multiple slides, different frags, split slides... 

`#+REVEAL: split`

**Comment by whilei** [comment]
Book titles/covers, lef/right brain diagrams, red/blue/gray-color/themeize(?)

**Comment by webappzero** [comment]
Yes, I like the idea of doing left brain right brain slides that correspond to the five stages of creativity. I thinned out her talk and have taken most of your recent suggestions, though I appreciated receiving all of them!

We could frag some book covers. That goes for any book in the talk, but we don't have to do every book. I guess its okay to show a book cover? never thought about that before regarding copyright. Hmmm.

Okay, after researching seems pubishers let you use covers for review, which is what I'm doing. Cool!
